Detectors/MUON/MID/Raw/test/testRaw.cxx

```diff
@@ -38,6 +38,13 @@
 
 BOOST_AUTO_TEST_SUITE(o2_mid_raw)
 
+o2::InteractionRecord getIR(uint64_t irLong)
```

Comment:
Instead of using map<uint64_t, ...> and then having to  
provide your own reverse operation from InteractionRecord::toLong() method, why don't you simply use maps where the IR is the key itself (as far as I can tell this works as InteractionRecord does implement the less than operator) ?

Reply:
Hi @aphecetche ,
good point: I've just changed it. Thanks!